### PR TITLE
feat: Link help center portal to an Inbox

### PR DESCRIPTION
**Pull request by tejaswinichile**
Fixes: https://linear.app/chatwoot/issue/CW-1470/link-an-inbox-to-a-help-center

https://www.loom.com/share/6df266e7350a426686eb5c5a47ed622e

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue [hunk 1]
```diff
@@ -259,6 +259,25 @@
...
+            selected-label
+            :select-label="$t('INBOX_MGMT.HELP_CENTER.SELECT_PLACEHOLDER')"
+            :deselect-label="$t('INBOX_MGMT.HELP_CENTER.REMOVE')"
+            :max-height="160"
+            :options="portal"
+            :allow-empty="true"
+            :option-height="104"
+          />
+        </div>
+
```
Comment (scmmishra) [inline app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue:280]: I don't think we need multiselect for this, we can just use a single select right?

**Review by scmmishra — CHANGES_REQUESTED** [review]
image

- [ ] Can we have a placeholder if no value is present?
- [ ] Also, we need to add a description like other fields
- [ ] The product has `Help Center` used everywhere, let's call it that in the copy, the code can have `portal`

File: app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue [hunk 2]
```diff
@@ -258,7 +258,22 @@
...
         </label>
-
+        <div class="medium-9 settings-item settings-item">
+          <label>
+            {{ $t('INBOX_MGMT.HELP_CENTER.LABEL') }}
+          </label>
+          <select v-model="selectedPortalSlug" class="filter__question">
+            <option value="" disabled>
+              {{ $t('INBOX_MGMT.HELP_CENTER.PLACEHOLDER') }}
+            </option>
```
Comment (scmmishra) [inline app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue:268]: At the moment, it's not possible to reset the value once set, we could do one of either things. Do we want to be able to reset it?

#### Add an option called "None" and hide the placeholder option

```suggestion
            <option value="" disabled hidden>
              {{ $t('INBOX_MGMT.HELP_CENTER.PLACEHOLDER') }}
            </option>
            <option value="">
              {{ $t('INBOX_MGMT.HELP_CENTER.NONE') }}
            </option>
```

#### Use the placeholder as the de-selected value

```suggestion
            <option value="">
              {{ $t('INBOX_MGMT.HELP_CENTER.PLACEHOLDER') }}
            </option>
```


#### Add a reset button

File: app/javascript/dashboard/i18n/locale/en/inboxMgmt.json [hunk 3]
```diff
@@ -482,6 +482,13 @@
       "WHATSAPP_WEBHOOK_SUBHEADER": "This token is used to verify the authenticity of the webhook endpoint.",
       "UPDATE_PRE_CHAT_FORM_SETTINGS": "Update Pre Chat Form Settings"
     },
+    "HELP_CENTER": {
+      "LABEL": "Help Center",
+      "PLACEHOLDER": "Select Help Center",
+      "SELECT_PLACEHOLDER": "Select Help Center",
+      "REMOVE": "Remove",
```
Comment (scmmishra) [inline app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:489]: Changes based on option 1 of [this suggestion](https://github.com/chatwoot/chatwoot/pull/6903/files#r1172809903)

```suggestion
      "REMOVE": "Remove Help Center ",
```

File: app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue (hunk 2 above)
Comment (tejaswinichile) [reply app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue:268]: @scmmishra  Having "None" value now to reset it, handling 'null' string in the backend, we will remove that line after fixing https://linear.app/chatwoot/issue/CW-1567/null-value-passed-as-null-string-to-backend

**Review by scmmishra — APPROVED** [review]

**Pull request merged**